lib/models/movie.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const Bookshelf = require('../libraries/bookshelf');
+
+module.exports = Bookshelf.Model.extend({
+  tableName: 'movies',
+  serialize: function () {
```

Comment:
Nit: Might want to standardize your tab sizes - in general we use 2 spaces for tabs.